torch/distributed/_tensor/experimental/attention.py

```diff
@@ -936,17 +992,23 @@ def unshard(
         return torch.cat(all_buffers, dim=seq_dim)
 
 
-class _StripeLoadBalancer(_LoadBalancer):
+class _RoundRobinLoadBalancer(_LoadBalancer):
     """
-    This load balancer chunk the buffer into cp_world_size * 2 shards, and
-    rank0 gets 0th and (cp_world_size * 2 - 1)th shards; aank1 will get 1st
-    and (cp_world_size * 2 - 2)th shards; ...
+    This load balancer chunk the buffer into cp_world_size * ROUND_ROBIN_CYCLE
+    shards, and uses a round robine approach to achieve load balancing.
+    Since ROUND_ROBIN_CYCLE begin 2 is enough for the causal masking, we currently
```

Comment:
typos (robine, begin 2)

clarity: not sure why "2 is enough for causal masking"?